storage/database/leveldb_database.go

```diff
@@ -78,14 +79,12 @@ func GetOpenFilesLimit() int {
 	if err != nil {
 		logger.Crit("Failed to retrieve file descriptor allowance", "err", err)
 	}
-	if limit < 2048 {
-		if err := fdlimit.Raise(2048); err != nil {
-			logger.Crit("Failed to raise file descriptor allowance", "err", err)
+	if limit < minFileDescriptors {
+		if err := fdlimit.Raise(minFileDescriptors); err != nil {
```

Comment:
How about removing calling `Raise` and make checkAndRaiseFDLimit only one to check and raise the file descriptors?

Reply:
I think it is not good idea to remove this raise and use `checkAndRaiseFDLimit` of RPC server which can be disabled.
How about finding better position?

Reply:
Then how about moving it under `SetKlayConfig`?